tests/e2e/kueue_test.go

```diff
@@ -164,6 +172,36 @@ func (tc *KueueTestCtx) testOwnerReferences() error {
 	return nil
 }
 
+func (tc *KueueTestCtx) validateVAPReady() error {
+	// if ocp is 4.17+ then VAP and VAPB should be created
+	if cluster.GetClusterInfo().Version.GTE(semver.MustParse("4.17.0")) {
+		keyvap := types.NamespacedName{Name: "kueue-validating-admission-policy"}
+		vap := &unstructured.Unstructured{}
+		vap.SetGroupVersionKind(gvk.ValidatingAdmissionPolicy)
+
+		err := tc.testCtx.customClient.Get(tc.testCtx.ctx, keyvap, vap)
```

Comment:
would be better to use gomega's `Eventually ` so that it won't fail in case the resource takes a little time to become available